docs/source/faq_sby.rst

```diff
@@ -17,3 +17,44 @@ statement behaves similar to an *asynchronous* reset that is not sampled
 by the clock, thus the sequence ``A && !B && !reset ##1 reset`` will disable
 the assumption, but will not disable the assertion in the above example.
 
+Choosing an engine and solver
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Q:** How do I know which formal engine and/or which solver to use for
+verifying my design?
+
+**A:** Different combinations of engine and solver will perform differently and
+may support different features or functionalities.  The `SBY engines reference
+<https://yosyshq.readthedocs.io/projects/sby/en/latest/reference.html#engines-section>`_
+lists all currently supported engines, their options, available solvers, and
+which modes they operate in.  At this time, a comprehensive comparison of when
+to use different engines/solvers is not yet available.
+
+With the introduction of the ``--autotune`` option it is now possible to
+automatically compare available engines for a given SBY task.  Autotune will run
+each engine, providing a report of time taken and the status returned.  This can
+be used to quickly determine the fastest configuration while continuing to
+iterate on a design. Check out the `autotune docs
+<https://yosyshq.readthedocs.io/projects/sby/en/latest/autotune.html>`_ for more
+information.
+
+Witness waves
```

Comment:
Calling this “Witness cover traces” would be more consistent with the terminology we use elsewhere